website/docs/r/tm_edge_cluster_qos.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_tm_edge_cluster_qos"
+sidebar_current: "docs-vcd-resource-tm-edge-cluster-qos"
+description: |-
+  Provides a VMware Cloud Foundation Tenant Manager Edge Cluster QoS resource.
+---
+
+# vcd\_tm\_edge\_cluster\_qos
+
+Provides a VMware Cloud Foundation Tenant Manager Edge Cluster QoS resource.
+
+-> This resource does not create an Edge Cluster QoS entity, but configures QoS for a given
+`edge_cluster_id`. Similarly, `terraform destroy` operation does not remove Edge Cluster, but resets
+QoS settings to default (unlimited). 
+
+## Example Usage
+
+```hcl
+data "vcd_tm_region" "demo" {
+  name = "region-one"
+}
+
+data "vcd_tm_edge_cluster" "demo" {
+  name             = "my-edge-cluster"
+  region_id        = data.vcd_tm_region.demo.id
+  sync_before_read = true
+}
+
+resource "vcd_tm_edge_cluster_qos" "demo" {
+  edge_cluster_id = data.vcd_tm_edge_cluster.demo.id
+
+  egress_committed_bandwidth_mbps  = 1
+  egress_burst_size_bytes          = 2
+  ingress_committed_bandwidth_mbps = 3
+  ingress_burst_size_bytes         = 4
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `edge_cluster_id` - (Required) An ID of Edge Cluster. Can be looked up using
+  [vcd_tm_edge_cluster](/providers/vmware/vcd/latest/docs/data-sources/tm_edge_cluster) data source
+* `egress_committed_bandwidth_mbps` - (Optional) Committed egress bandwidth specified in Mbps.
+  Bandwidth is limited to line rate. Traffic exceeding bandwidth will be dropped
+* `egress_burst_size_bytes` - (Optional) Egress burst size in bytes
+* `ingress_committed_bandwidth_mbps` - (Optional) Committed ingress bandwidth specified in Mbps.
+  Bandwidth is limited to line rate. Traffic exceeding bandwidth will be dropped
+* `ingress_burst_size_bytes` - (Optional) Ingres burst size in bytes
+
+-> Choosing to set _egress_ or _ingress_ requires both of that traffic direction must be set
```

Comment:
Would you rephrase this, hard to grasp right now :) Maybe change "egress" & "ingress" to specific parameters and then review the ending about the direction.

Reply:
I had my doubts about clarity when writing this - apparently it wasn't too good. I changed the approach and mentioning requirements inline for each field. (there are also validations applied in the schema, so user will be hinted)

<img width="706" alt="image" src="https://github.com/user-attachments/assets/99f6f351-cabd-4df7-9718-e70c1f43359f" />

Reply:
Perfect! Reported a typo :)